src/views/admin/AdminView.tsx

```diff
@@ -23,20 +25,72 @@ export const AdminView = () => {
       window.location.reload()
     }
   }
+
+  const handleExportClick = () => {
+    const storageKeys = Object.keys(window.localStorage)
+    const storage = storageKeys.reduce((acc, key) => {
+      const rawValue = window.localStorage.getItem(key)
+      if (rawValue) {
+        acc[key] = JSON.parse(rawValue)
+      }
+
+      return acc
+    }, {} as Record<string, unknown>)
+    const jsonStorage = JSON.stringify(storage)
+
+    const linkElement = document.createElement('a')
+    linkElement.setAttribute('href', 'data:application/json;charset=utf-8,' + encodeURIComponent(jsonStorage))
+    linkElement.setAttribute('download', `atlas-export-${new Date().toLocaleString()}.json`)
```

Comment:
This will produce a file name without spaces and forbidden characters (`:`)

```suggestion
    linkElement.setAttribute('download', `atlas-export-${new Date().toISOString()}.json`)
```